src/ralph/backends/database/es.py

```diff
@@ -0,0 +1,66 @@
+"""Elasticsearch storage backend for Ralph"""
+
+import json
+import logging
+import sys
+
+from elasticsearch import Elasticsearch
+from elasticsearch.helpers import scan, streaming_bulk
+
+from .base import BaseDatabase
+
+logger = logging.getLogger(__name__)
+
+
+class ESDatabase(BaseDatabase):
+    """Elasticsearch database backend"""
+
+    name = "es"
+
+    def __init__(self, hosts, index):
+        """Instanciate the Elasticsearch client.
+
+        hosts is supposed to be a list
+
+        """
+
+        self._hosts = hosts
```

Comment:
self._hosts is not used ?
```suggestion

```

Reply:
You are right, I should use it to instantiate the Elasticsearch client.